lighthouse-core/audits/dobetterweb/inspector-issues.js

```diff
@@ -71,19 +71,19 @@ class IssuesPanelEntries extends Audit {
   }
 
   /**
-   * @param {Array<LH.Crdp.Audits.SameSiteCookieIssueDetails>} sameSiteCookieIssues
+   * @param {Array<LH.Crdp.Audits.CookieIssueDetails>} CookieIssues
    * @return {LH.Audit.Details.TableItem}
    */
-  static getSameSiteCookieRow(sameSiteCookieIssues) {
+  static getCookieRow(CookieIssues) {
     const requestUrls = new Set();
-    for (const issue of sameSiteCookieIssues) {
+    for (const issue of CookieIssues) {
       const requestUrl = (issue.request?.url) || issue.cookieUrl;
       if (requestUrl) {
         requestUrls.add(requestUrl);
       }
     }
     return {
-      issueType: 'SameSite cookie',
+      issueType: ' cookie',
```

Comment:
`s/samesite//gI`?

I think this needs to be `'Cookie'`

Reply:
lol